django_enumfield/enum.py

```diff
@@ -23,6 +24,9 @@ def label(self):
 class Enum(NativeEnum):
```

Comment:
I've tried subclassing from `IntEnum` and the tests pass, regardless of the version: https://travis-ci.org/5monkeys/django-enumfield/builds/86409961